app/controllers/tagging_spreadsheets_controller.rb

```diff
@@ -15,7 +15,7 @@ def create
     if tagging_spreadsheet.valid?
       tagging_spreadsheet.save!
       InitialTaggingImport.perform_async(tagging_spreadsheet.id)
-      redirect_to tagging_spreadsheets_path
+      redirect_to tagging_spreadsheet, success: "Spreadsheet uploaded. It will be imported within a few minutes."
```

Comment:
What do you think of moving strings like these to I18n (not for translation purposes, just to avoid having long strings)?